extension/chrome/settings/modules/contacts.ts

```diff
@@ -109,26 +110,32 @@ Catch.try(async () => {
 
   const actionProcessBulkImportTextInput = async () => {
     try {
-      const value = String($('#bulk_import .input_pubkey').val());
+      const value = Str.normalize(String($('#bulk_import .input_pubkey').val())).trim();
       const normalizedLongid = KeyImportUi.normalizeLongId(value);
-      let pubkey: string;
+      let key: string;
       if (normalizedLongid) {
         const data = await Keyserver.lookupLongid(acctEmail, normalizedLongid);
         if (data.pubkey) {
-          pubkey = data.pubkey;
+          key = data.pubkey;
         } else {
           await Ui.modal.warning('Could not find any Public Key in our public records that matches this fingerprint or longid');
           return;
         }
       } else {
-        pubkey = value;
+        key = value;
       }
-      const replacedHtmlSafe = XssSafeFactory.replaceRenderableMsgBlocks(factory, pubkey);
-      if (replacedHtmlSafe && replacedHtmlSafe !== value) {
-        $('#bulk_import #processed').html(replacedHtmlSafe).css('display', 'block'); // xss-safe-factory
-        $('#bulk_import .input_pubkey, #bulk_import .action_process, #file_import #fineuploader_button').css('display', 'none');
+      if (Pgp.key.isPossiblePublic(key)) {
```

Comment:
You don't have to add these `isPossible` methods. You can instead just get the `MsgBlock[]`, and then filter the blocks, replacing all that is not a public key with a note `Skipping found XXX`, where XXX is the name of the block you don't like.

That way you still render the public keys, and there will be notes around them that the other things were skipped (not modals)